primitives/trie/src/lib.rs

```diff
@@ -119,6 +121,51 @@ pub mod trie_types {
 	pub type TrieError<H> = trie_db::TrieError<H, super::Error>;
 }
 
+/// Create a proof for a subset of keys in a trie.
+///
+/// The `keys` may contain any set of keys regardless of each one of them is included
+/// into the `db`.
+///
+/// For a key `K` that is included into the `db` a proof of inclusion is generated.
+/// For a key `K` that is not included into the `db` a proof of non-inclusion is generated.
+/// These can be later checked in [`verify_trie_proof`].
+///
+/// [`verify_trie_proof`]: ./fn.verify_trie_proof.html
```

Comment:
```suggestion
```

Not understanding the reason for this line.

You already link the function above.